CASMNET-1809 remove mgmt net upgrade docs (#2307)
Cray-HPE/docs-csm: upgrade/README.md

See [Prepare for Upgrade](prepare_for_upgrade.md).

## 2. (Optional) Enable Customer High Speed Network Routing

If the CHN is the desired network for customer access, see [Enable Customer High Speed Network Routing](../operations/network/management_network/bican_enable.md).

> This should be completed before the management nodes are upgraded so that the correct HSN IP addresses are allocated and applied during the upgrade.

Reviewer: ```suggestion
If the CHN configuration is not in place before the NCN nodes are upgraded, the NCNs will not receive proper HSN IP addresses and customer access will be broken.
```

> This feature has additional requirements which include.

Reviewer: ```suggestion
The CHN feature also has additional requirements beyond the system configuration changes, and include
```

> - Customer edge switches

Reviewer: ```suggestion
- Customer edge switch configuration updates
```

> - Cabling from HSN to edge switches

Reviewer: ```suggestion
- Validation of cabling from HSN to edge switches
```

> - Additional IP address space

Reviewer: ```suggestion
- Additional IP address space configuration
```
